src/main/java/com/denizenscript/depenizen/bukkit/clientizen/ClientizenSupport.java

```diff
@@ -13,49 +13,41 @@
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.Listener;
 import org.bukkit.event.player.PlayerQuitEvent;
-import org.bukkit.plugin.messaging.PluginMessageListener;
-import org.jetbrains.annotations.NotNull;
 
 import java.io.File;
 import java.io.FileInputStream;
 import java.util.*;
 
-public class ClientizenSupport implements Listener, PluginMessageListener {
+public class ClientizenSupport implements Listener {
 
     public static ClientizenSupport instance;
 
     public static final Map<String, String> clientizenScripts = new HashMap<>();
+    public static DataSerializer scriptsSerializer;
     public static final Set<UUID> clientizenPlayers = new HashSet<>();
 
-    public static final File clientizenFolder;
-    public static final String CHANNEL_NAMESPACE = "clientizen";
+    public static File clientizenFolder;
 
-    static {
-        clientizenFolder = new File(Denizen.getInstance().getDataFolder(), "client-scripts");
-        clientizenFolder.mkdirs();
-    }
-
-    public ClientizenSupport() {
-        Bukkit.getPluginManager().registerEvents(this, Depenizen.instance);
+    public static void init() {
+        instance = new ClientizenSupport();
+        clientizenFolder = new File(Denizen.instance.getDataFolder(), "client-scripts");
+        clientizenFolder.mkdir();
+        Bukkit.getPluginManager().registerEvents(instance, Depenizen.instance);
+        // A tag for testing
```

Comment:
This isn't just a tag for testing. It's important to distinguish players as Clientizen

Reply:
It's a tag for testing in the sense that this is not where it would actually go, it's currently there for us to test with and will be moved to a more proper location once we add server-side Clientizen related tags - basically that's just `ignore the fact that this is in the wrong place`